pkg/scrape/target_test.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2024 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scrape
+
+import (
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/parca-dev/parca/pkg/config"
```

Comment:
the linter is complaining that this import needs to be separated from the "external" imports

Reply:
fair. let me go over the pre-commit hooks you got here. i should've caught these earlier.